### DISCUSSION TITLE
[Question] draw.io Feature question

### DISCUSSION THREAD

**Discussion by Chitsol** [jgraph/drawio#3866] [General]
When I click on object A within the same page, is there a function to change the screen to where object B is located or to float information?

**Comment by Kubanis** [comment]
> When I click on object A within the same page, is there a function to change the screen to where object B is located or to float information?

It is possible - you have to create a custom link with a little Json scrip which uses "select" action to select and scroll to the specific shape based on it's ID or tag.
Here you can read more detailed info
https://www.drawio.com/doc/faq/custom-links
